### PR TITLE
Add Praat scripting language

### PR DESCRIPTION
**Pull request by JJWRoeloffs**
Adding support for the Praat scripting language. This is a domain-specific language that is quite commonly used in academia for linguistic research, but that isn't that commonly known outside that sphere.

I am not affiliated with the project - I'm just a student who had to learn the language for a BA course, and was surprised github did not support it.

## Description
If I did everything correctly, this commit adds support for the Praat scripting language in the way described by `CONTRIBUTING.md`

## Checklist:
- [x] **I am adding a new language.**
  - [x] The extension of the new language is used in hundreds of repositories on GitHub.com.
    - Search results for each extension (2.5K files):
      -  https://github.com/search?type=code&q=NOT+is%3Afork+path%3A*.praat
      -  I also added `praatscript`, `praat_script` and `praat-script`, which are less frequently used, but won't likely overlap with anything. I included them because some more common grammars include them [example1](https://github.com/mauriciofigueroa/praatSublimeSyntax/blob/0077112a7435084f6281de03d16de894a8ecc1bf/Praat.tmLanguage#L4) [example2](https://github.com/massCodeIO/massCode/blob/ae9eb6b2569357eadd27e363dd403f5b24b63c2d/src/renderer/components/editor/grammars/praat.tmLanguage.json#L4)
      - I'll also list a few projects using this language, alongside their star count:
        - [FastTrack](https://github.com/santiagobarreda/FastTrack) (51) - a praat plugin
        - [SpeCT](https://github.com/lennes/spect) (49) - Speech Corpus Toolkit for Praat
        - [Praatsouce](https://github.com/kirbyj/praatsauce) (21) - Praat-based tools for spectral analysis
        - [feelins/PraatScripts](https://github.com/feelins/Praat_Scripts) (169) - A sample repository containing example scripts
        - [stylerw's praat scripts](https://github.com/stylerw/styler_praat_scripts) (87) - A repository with the praat scripts this person finds useful for research.
        - [fieldDB's praat scripts](https://github.com/FieldDB/Praat-Scripts) (50) - Another repository of scripts some researcher found useful.
  - [x] I have included a real-world usage sample for all extensions added in this PR:
    - Sample license(s):
      - I wrote these scripts myself as a part of a Research inter-ship, and am happy to re-licence them under linguist's MIT licence (which I am allowed to do following the terms of the internship).
  - [x] I have included a syntax highlighting grammar: [URL to grammar repo](https://github.com/JJWRoeloffs/Praat-tmLanguage)
  - [x] I have added a color
    - Hex value: `#c8506d`
    - Rationale: Colour from the logo. The logo has a small gradient, so I picked the colour with the highest contrast out of the gradient to both black and white.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -1335,3 +1335,6 @@
 [submodule "vendor/grammars/zephir-sublime"]
 	path = vendor/grammars/zephir-sublime
 	url = https://github.com/phalcon/zephir-sublime
+[submodule "vendor/grammars/Praat-tmLanguage"]
+	path = vendor/grammars/Praat-tmLanguage
+	url = https://github.com/JJWRoeloffs/Praat-tmLanguage.git
```
Comment (lildude) [inline .gitmodules:1340]: Why not use the upstream grammar? Linguist has no problems with VSCode grammars as they're TextMate compatible.

File: lib/linguist/languages.yml [hunk 2]
```diff
@@ -5328,6 +5328,19 @@ PowerShell:
...
+  color: "#c8506d"
+  tm_scope: source.praat
+  ace_mode: praat
+  extensions:
+  - ".praat"
+  - ".praat-script"
+  - ".praat_script"
+  - ".praatscript"
+  interpreters:
+  - praat
```
Comment (lildude) [inline lib/linguist/languages.yml:5342]: We only need the interpreter if the language is commonly used in shell scripts and this has a hashbang line at the very beginning.

Comment (JJWRoeloffs) [reply lib/linguist/languages.yml]: I see, I removed that now.

To be honest: I was under the impression that the shebang was supported but simply underused (Most users are on Windows, and double-click the file in explorer), but, after testing, I can confirm it doesn't work with the current latest version of Praat.

**Review by lildude — CHANGES_REQUESTED** [review]
Please remove vendor/grammars/Praat-tmLanguage

**Comment by JJWRoeloffs** [comment]
Checked everything & updated with master. I think everything is ready at this point.

**Review by lildude — APPROVED** [review]

**Pull request merged**